src/sagemaker/fw_utils.py

```diff
@@ -59,7 +59,7 @@
 OPT_IN_ACCOUNTS_BY_REGION = {"ap-east-1": "057415533634", "me-south-1": "724002660598"}
 ASIMOV_OPT_IN_ACCOUNTS_BY_REGION = {"ap-east-1": "871362719292", "me-south-1": "217643126080"}
 DEFAULT_ACCOUNT = "520713654638"
-ASIMOV_PROD_ACCOUNT = "763104351884"
+ASIMOV_PROD_ACCOUNT = "028651357192" if os.getenv("DLC_GAMMA_TESTING") else "763104351884"
 ASIMOV_DEFAULT_ACCOUNT = ASIMOV_PROD_ACCOUNT
```

Comment:
I think it makes more sense to preserve `ASIMOV_PROD_ACCOUNT` as the prod account number

```suggestion
ASIMOV_PROD_ACCOUNT = "763104351884"
ASIMOV_DEFAULT_ACCOUNT = "028651357192" if os.getenv("DLC_GAMMA_TESTING") else ASIMOV_PROD_ACCOUNT
```